### PR TITLE
Fix examples definition, make it consistent

### PR DESCRIPTION
**Pull request by chenliang613**
1.Define an unified object to create carboncontext.
2.For different examples, fix inconsistent definition.

### CODE REVIEW

**Comment by chenliang613** [comment]
retest this please

File: examples/src/main/scala/org/carbondata/examples/CarbonExample.scala [hunk 1]
```diff
@@ -17,42 +17,18 @@
...
 
-    // As Carbon using kettle, so need to set kettle configuration
-    cc.setConf("carbon.kettle.home", kettleHome)
-    cc.setConf("hive.metastore.warehouse.dir", hiveMetaPath)
-
-    // whether use table split partition
-    // true -> use table split partition, support multiple partition loading
-    // false -> use node split partition, support data load by host partition
-    CarbonProperties.getInstance().addProperty("carbon.table.split.partition.enable", "false")
+    val cc = InitForExamples.createCarbonContext(sc)
```
Comment (ravipesala) [inline examples/src/main/scala/org/carbondata/examples/CarbonExample.scala]: I guess better create `SparkContext` also inside `InitForExamples.createCarbonContext` by passing appName to it.

Comment (chenliang613) [inline examples/src/main/scala/org/carbondata/examples/CarbonExample.scala]: sounds good, will modify it.

**Comment by chenliang613** [comment]
retest this please

**Comment by ravipesala** [comment]
LGTM

**Pull request merged**